plugins/flytekit-modin/setup.py

```diff
@@ -0,0 +1,34 @@
+from setuptools import setup
+
+PLUGIN_NAME = "modin"
+
+microlib_name = f"flytekitplugins-{PLUGIN_NAME}"
+
+plugin_requires = ["flytekit>=0.22.0,<1.0.0", "modin", "ray"]

```

Comment:
do you guys need ray? i thought ray is optional? 
Because if you need ray, then is it only in local mode?

Reply:
it is either Ray or Dask., the other backends are still experimental. Yes, local mode

Reply:
Got it 👍🏼